src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/CommonFunction.java

```diff
@@ -116,6 +117,8 @@ public static boolean isContestedApplication(CaseDetails caseDetails) {
     }
 
     public static boolean isOrderApprovedDocumentCollectionPresent(Map<String, Object> caseData) {
-        return caseData.get(APPROVED_ORDER_COLLECTION) != null && !((List<Map>) caseData.get(APPROVED_ORDER_COLLECTION)).isEmpty();
+        return (caseData.get(APPROVED_ORDER_COLLECTION) != null && !((List<Map>) caseData.get(APPROVED_ORDER_COLLECTION)).isEmpty())
+            || (caseData.get(CONTESTED_CONSENT_ORDER_COLLECTION) != null
+            && !((List<Map>) caseData.get(CONTESTED_CONSENT_ORDER_COLLECTION)).isEmpty());
```

Comment:
can you move the new lines out to an appropriately named private method and call it from here, I think there might already be a function that will check for a collection that you can re-use